app/assets/javascripts/darkswarm/controllers/group_page_controller.js.coffee

```diff
@@ -1,10 +1,8 @@
-Darkswarm.controller "GroupPageCtrl", ($scope, group_enterprises, Enterprises, MapConfiguration, OfnMap, visibleFilter, Navigation) ->
+Darkswarm.controller "GroupPageCtrl", ($scope, enterprises, Enterprises, MapConfiguration, OfnMap, visibleFilter, Navigation) ->
   $scope.Enterprises = Enterprises
 
-  all_enterprises_by_id = Enterprises.enterprises_by_id
-
-  dereferenced_enterprises = group_enterprises.map (enterprise) =>
-    all_enterprises_by_id[enterprise.id]
+  dereferenced_enterprises = enterprises.map (enterprise) =>
+    Enterprises.enterprises_by_id[enterprise.id]
```

Comment:
you have removed the dereferencing right? so these enterprises are not dereferenced.
I guess you can change the name of this dereferenced_enterprises variable.

Reply:
Right, there's still some dereferencing going on in some connected places, but this part is now actually just `enterprises_by_id`. I'll update it.